tests/ontopy_tests/test_patch.py

```diff
@@ -87,6 +92,13 @@ def test_get_by_label_onto(emmo: "Ontology") -> None:
     )  # Check that wikipediaReference can be acceses as attribute
 
 
+def test_get_indirect_is_a(emmo: "Ontology") -> None:
+    assert any(
+        re.match("^emmo.*.hasDimensionString.value(.*)$", str(e))
+        for e in emmo.MicroPascal.get_indirect_is_a()
+    )
+
+
```

Comment:
Why is this test put in test_get_by_label?

Reply:
Its not, it is put in test_patch.py